src/es-dev-server-rollup.ts

```diff
@@ -64,15 +58,15 @@ export function wrapRollupPlugin(
         return;
       }
 
-      if (whatwgUrl.parseURL(source) != null) {
-        // don't resolve valid urls
+      if (! path.isAbsolute(source) && whatwgUrl.parseURL(source) != null) {
+        // don't resolve relative and valid urls
         return source;
       }
 
       const requestedFile = context.path.endsWith('/')
         ? `${context.path}index.html`
         : context.path;
-      const filePath = resolveFilePath(rootDir, requestedFile);
+      const filePath = path.join(rootDir, requestedFile);
```

Comment:
Doesn't this do something like this on windows:

```js
path.join('C:\\foo\bar', './lorem/ipsum.js')
```

Reply:
Yes, to be exact: 
```javascript
path.join('C:\\foo\\bar', './lorem/ipsum.js')
```
This will result into:
```
'c:\\foo\\bar\\lorem\\ipsum.js'
```
Which is what we are looking for, afaik. This is because `path.join` does a `path.normalize` at the end, which will normalize the full path according to the current OS (https://nodejs.org/api/path.html#path_path_join_paths)